### ISSUE TITLE
[BUG][NOWEB] - Webhook when viewing status not working

### ISSUE THREAD

**Issue by MoishyShuvax** [allburov/whatsapp-http-api#329]
**Webhook when viewing status not working**

**Version**

```json
{
  "version": "2024.5.6",
  "engine": "NOWEB",
  "tier": "PLUS",
  "browser": "/usr/bin/chromium"
}
```

**Expected behavior**

While waiting for the `message.ack` event, a webhook should be sent whenever a contact views the status I sent

In practice, the webhook is received only in the first view and not in the following views

**Comment by devlikepro** [comment]
>In practice, the webhook is received only in the first view and not in the following views

That's right, it works this way even in the origial WA app - if you watch status multiple times **from the same account** - it shows "1 view(s)"
If you watch it **from different account** - it'll send you a new `message.ack` for that view (if the account has the right "send ack" settings)

let us know if we misunderstood smth here!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by MoishyShuvax** [comment]
No, I'm talking about viewing from another account
The webhook is sent only on the first view, and subsequent views from other accounts are not sent

**Comment by oteGroupMoishy** [comment]
> No, I'm talking about viewing from another account
> The webhook is sent only on the first view, and subsequent views from other accounts are not sent

@devlikepro Any update on this?

**Comment by allburov** [comment]
Working on that! Found the root cause, fixing it in the engine, hopefully fix will be available soon! :crossed_fingers: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Fixed in `2024.5.14`!
Also fixed `from/to` name - it'll be `status@broadcast`

```json
{
    "event": "message.ack",
    "session": "default",
    "me": {
        "id": "123123@c.us",
        "pushName": "Sim1"
    },
    "payload": {
        "id": "true_status@broadcast_AAAAAAAAAAAAAAAAA",
        "from": "status@broadcast",
        "to": "status@broadcast",
        "participant": "321321@c.us",
        "fromMe": true,
        "ack": 3,
        "ackName": "READ"
    },
    "engine": "NOWEB",
    "environment": {
        "version": "2024.5.14",
        "engine": "NOWEB",
        "tier": "PLUS",
        "browser": "/usr/bin/google-chrome-stable"
    }
}
```

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
